llvm/include/llvm/SandboxIR/SandboxIR.h

```diff
@@ -1451,6 +1456,63 @@ class ShuffleVectorInst final
   }
 };
 
+class InsertValueInst
+    : public SingleLLVMInstructionImpl<llvm::InsertValueInst> {
+  /// Use Context::createInsertValueInst(). Don't call the constructor directly.
+  InsertValueInst(llvm::InsertValueInst *IVI, Context &Ctx)
+      : SingleLLVMInstructionImpl(ClassID::InsertValue, Opcode::InsertValue,
+                                  IVI, Ctx) {}
+  friend Context; // for InsertValueInst()
+
+public:
+  static Value *create(Value *Agg, Value *Val, ArrayRef<unsigned> Idxs,
+                       BBIterator WhereIt, BasicBlock *WhereBB, Context &Ctx,
+                       const Twine &Name = "");
+
+  using idx_iterator = llvm::InsertValueInst::idx_iterator;
```

Comment:
No
```
static bool classof(const Value *From) {
    return From->getSubclassID() == ClassID::InsertValueInst;
}
```

Reply:
@tschuett good catch, thanks!

@vporpo It does pass, and I just double-checked I'm running them with asserts enabled (debug build).

I think it's falling back to `Instruction::classof` and accidentally passing.

Reply:
I wonder if there is a good way to check that classof has been implemented. Perhaps a test that uses the .def file and  does something with `<subclass>::classof()` so that we get a compile-time error if we forget to implement it.

Reply:
I wrote a small test that checks `EXPECT_NE(&<subclass>::classof, &Instruction::classof);` and one of the insturctions is missing a classof!

Reply:
Great! Can you commit that test (and the missing classof) separately from this PR?

Reply:
Yes I will upload a PR in a few minutes.